packages/x-components/src/components/__tests__/base-variable-column-grid.spec.ts

```diff
@@ -12,63 +12,64 @@ const itemsStub = [
   ...searchResponse.results
 ];
 
-function renderComponent({ items = itemsStub }: RenderOptions): RenderAPI {
+function renderComponent({ items = itemsStub }: RenderOptions = {}): RenderAPI {
   const [, localVue] = installNewXPlugin();
-  const wrapper = mount(BaseVariableColumnGrid, {
-    components: {
-      BaseVariableColumnGrid
```

Comment:
Just for my information. This was wrong, wasn't it? I mean importing that component which is not used in the template.

Reply:
Yep, the `components` option is used to locally register components, so then in the render function or in the template you can find them by their name. But this was directly mounting the `BaseVariableColumnGrid` component, there was no custom wrapper component like we do in other cases. So it is unnecessary.